README.md

```diff
@@ -14,6 +14,9 @@ And it runs on all platforms Rust supports, including Windows.
 [rustlang]: https://www.rust-lang.org
 
 * [Installation](#installation)
+  * [Profiles](#profiles)
```

Comment:
Indentation gone wrong

Reply:
No, it is intentional, these are subsections. But maybe we don't want them.

Reply:
I think the subsections are okay.